string-manipulation-2/src/swap-chars.js

```diff
@@ -0,0 +1,44 @@
+/* exported swapChars */
+function swapChars(firstIndex, secondIndex, string) {
+  const a = [];
+  let s = '';
+  for (let i = 0; i < string.length; i++) {
+    if (i === firstIndex) {
+      a.push(string[secondIndex]);
+    } else if (i === secondIndex) {
+      a.push(string[firstIndex]);
+    } else {
+      a.push(string[i]);
+    }
+  } for (let i = 0; i < a.length; i++) {
```

Comment:
nice job figuring this out! 
I think you could do this with just the `s` var and not using the `a` array.

Instead of `push`ing to the `a` array, you could just to a string concatenation (`s += string[index]`) in each of your conditional statements.

That is basically what you are doing on line 14 in a separate `for` loop. 👍